btcjson/chainsvrcmds.go

```diff
@@ -333,6 +333,19 @@ func NewGetInfoCmd() *GetInfoCmd {
 	return &GetInfoCmd{}
 }
 
+// GetMempoolEntryCmd defines the getmempoolentry JSON-RPC command.
+type GetMempoolEntryCmd struct {
+	TxID string
```

Comment:
doesn't need it actually. json-rpc requests are positional, not keyed in an object.